src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -29,24 +29,118 @@
 import org.spongepowered.api.block.BlockType;
 import org.spongepowered.api.block.BlockTypes;
 import org.spongepowered.api.block.tileentity.CommandBlock;
-import org.spongepowered.api.data.manipulator.mutable.*;
-import org.spongepowered.api.data.manipulator.mutable.block.*;
-import org.spongepowered.api.data.manipulator.mutable.entity.*;
-import org.spongepowered.api.data.manipulator.mutable.item.*;
```

Comment:
Eclipse formatter decided to do this. Trying to figure out how to prevent it from doing so.